samconfig.toml

```diff
@@ -0,0 +1,34 @@
+# More information about the configuration file can be found here:
```

Comment:
do we want to commit this file or let the users generate their own with the following command?

```bash
sam deploy --guided
```